deploy/csi-hostpath-plugin.yaml

```diff
@@ -64,7 +64,7 @@ spec:
           # for development purposes; eventually switch to IfNotPresent
           imagePullPolicy: Always
           command:
-            - csi-driver-projected-resource
+            - csi-driver-shared-resource
           args:
             - "--drivername=csi.shared-resources.openshift.io"
```

Comment:
Is this correct? Should it be singular?

Reply:
Yes - it is singular. More renames coming to line it all up (as you can see, we have other bits that call it "shared-resources".